### PR TITLE
Add breadcrumb navigation on the Dashboard > Firm > Edit screen

### PR DESCRIPTION
**Pull request by neoeno**

### CODE REVIEW

File: app/views/shared/_breadcrumbs.html.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <li class="breadcrumbs__item" itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
           <span class="visually-hidden">&gt;</span>
             <span itemprop="title">
-              <%= t(last_link[:locale_key]) %>
+              <%= (last_link[:locale_key] && t(last_link[:locale_key])) || last_link[:name] %>
```
Comment (neoeno) [inline app/views/shared/_breadcrumbs.html.erb]: This is a little hard to parse — could we make it clearer? maybe through a helper if nothing else makes sense

File: app/views/shared/_breadcrumbs.html.erb [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <li class="breadcrumbs__item" itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
           <span class="visually-hidden">&gt;</span>
             <span itemprop="title">
-              <%= t(last_link[:locale_key]) %>
+              <%= last_link[:name] || t(last_link[:locale_key]) %>
```
Comment (munckymagik) [inline app/views/shared/_breadcrumbs.html.erb]: @neoeno better?

Comment (neoeno) [inline app/views/shared/_breadcrumbs.html.erb]: better

**Comment by neoeno** [comment]
:+1:

**Comment by tomas-stefano** [comment]
LGTM :+1:
